### PR TITLE
Fix/pointer visibility

### PR DESCRIPTION
**Pull request by thestonefox**

### CODE REVIEW

**Comment by thestonefox** [comment]
This is a prerequisite for the destination point work. Without working pointer logic, the dest points won't work either.

There is another commit on VRTK repo that relies on this zinnia PR being merged first too.

File: Runtime/Visual/PointsRenderer.cs [hunk 1]
```diff
@@ -27,18 +27,36 @@ public class PointsData
             [field: DocumentedByXml]
             public GameObject StartPoint { get; set; }
             /// <summary>
+            /// Whether the first point should be visible;
```
Comment (bddckr) [inline Runtime/Visual/PointsRenderer.cs]: `;` -> `.`

File: Runtime/Visual/PointsRenderer.cs [hunk 2]
```diff
@@ -27,18 +27,36 @@ public class PointsData
...
+            [field: DocumentedByXml]
+            public bool IsStartPointVisible { get; set; }
+            /// <summary>
             /// Represents the segments between <see cref="Start"/> and <see cref="End"/>. This will get cloned to create all the segments.
             /// </summary>
             [Serialized]
             [field: DocumentedByXml]
             public GameObject RepeatedSegmentPoint { get; set; }
             /// <summary>
+            /// Whether the repeated segment point(s) should be visible;
```
Comment (bddckr) [inline Runtime/Visual/PointsRenderer.cs]: `;` -> `.`

File: Runtime/Visual/PointsRenderer.cs [hunk 3]
```diff
@@ -27,18 +27,36 @@ public class PointsData
...
+            [field: DocumentedByXml]
+            public bool IsRepeatedSegmentPointVisible { get; set; }
+            /// <summary>
             /// Represents the end, i.e. the last rendered point.
             /// </summary>
             [Serialized]
             [field: DocumentedByXml]
             public GameObject EndPoint { get; set; }
             /// <summary>
+            /// Whether the end point should be visible;
```
Comment (bddckr) [inline Runtime/Visual/PointsRenderer.cs]: `;` -> `.`

**Pull request merged**